src/SyntheticDataWorkValidator.sol

```diff
@@ -84,6 +86,37 @@ contract SyntheticDataWorkValidator is IWorkValidation {
         return (info.provider, info.nodeId);
     }
 
+    function softInvalidateWork(uint256 poolId, bytes calldata data) external returns (address, address, uint256) {
+        require(msg.sender == computePool, "Unauthorized");
+        require(data.length >= 64, "Data too short for soft invalidation");
```

Comment:
why is this `>= 64` when `invalidateWork` just requires `>= 32` and we don't seem to need to read past 64 bits.

Reply:
ah nevermind this is right I don't know what I was thinking lol